src/Microsoft.SqlTools.ServiceLayer/Localization/sr.strings

```diff
@@ -200,6 +200,8 @@ EditDataScriptFilePathNull = An output filename must be provided
 
 EditDataCommitInProgress = A commit task is in progress. Please wait for completion.
 
+EditDataComputedColumnPlaceholder = <TBD>
```

Comment:
SSMS - Doesn't show defaults. Just fills all fields in with "NULL".
SSDT - Doesn't show defaults. Just fills all fields in with "NULL".
JetBrains - Uses "<Generated>" as the place holder. Fields with defaults are shown as "<Default>"